### PR TITLE
feat: style session forms and add navbar

### PR DESCRIPTION
**Pull request by Theosaurus-Rex**
We have a navbar and stuff!
Styles a functional navbar and the login/registration forms
Screen Shot 2022-08-16 at 10 21 31 am
Screen Shot 2022-08-16 at 10 21 38 am

### CODE REVIEW

File: test/playdate_web/controllers/page_controller_test.exs [hunk 1]
```diff
@@ -1,8 +1,3 @@
 defmodule PlaydateWeb.PageControllerTest do
   use PlaydateWeb.ConnCase
```
Comment (juddey) [inline test/playdate_web/controllers/page_controller_test.exs:2]: I think it's probably worth leaving a test in here if we are going to use a static controller to host a front page. You can skip tests with  `@tag :skip` If you just  want to disable them temporarily :)
Not sure of the intent though, feel free to bin  if it's not appropriate

File: lib/playdate_web/templates/user_reset_password/edit.html.heex [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
 <h5 class="px-5 pb-2 text-lg leading-normal text-center">Reset Password</h5>
 <%= form_for @form, Routes.user_reset_password_path(@conn, :update, @token), fn f -> %>
   <%= if @form.submitted_once? do %>
```
Comment (juddey) [inline lib/playdate_web/templates/user_reset_password/edit.html.heex:4]: I'm not sure about `@form.submitted_once` - doesn't seems to be referenced anywhere?

**Review by juddey — APPROVED** [review]

File: lib/playdate_web/templates/user_reset_password/edit.html.heex (hunk 2 above)
Comment (jimsynz) [reply lib/playdate_web/templates/user_reset_password/edit.html.heex:4]: @juddey these came from the ash authentication example that I copy pasta'd.

File: test/playdate_web/controllers/page_controller_test.exs (hunk 1 above)
Comment (jimsynz) [reply test/playdate_web/controllers/page_controller_test.exs:2]: We switched to a live view for the home page, so we don't need to the controller or anything.

**Comment by juddey** [comment]
Approved pending the `submitted_once` question, @Theosaurus-Rex

**Comment by jimsynz** [comment]
Please also squash into a conventional commit before merging.

File: test/playdate_web/controllers/page_controller_test.exs (hunk 1 above)
Comment (Theosaurus-Rex) [reply test/playdate_web/controllers/page_controller_test.exs:2]: I'll delete the whole controller @jimsynz ?

File: lib/playdate_web/templates/user_reset_password/edit.html.heex (hunk 2 above)
Comment (Theosaurus-Rex) [reply lib/playdate_web/templates/user_reset_password/edit.html.heex:4]: @juddey This is coming from AshPhoenix.Form, it lets us show an alert in the case that form submission fails

File: test/playdate_web/controllers/page_controller_test.exs (hunk 1 above)
Comment (jimsynz) [reply test/playdate_web/controllers/page_controller_test.exs]: :+1:

**Pull request merged**